distributed/worker.py

```diff
@@ -700,6 +702,10 @@ def _register_with_scheduler(self):
                         continue
                     future = gen.with_timeout(timedelta(seconds=diff), future)
                 response = yield future
+
+                if response.get("warning"):
+                    warnings.warn(response["warning"])
```

Comment:
Does distributed enable `logging.captureWarnings`? Otherwise should this potentially be reported using `logging` instead?

Reply:
Sure.  Sounds good.